src/nodes/VariableDeclaration.js

```diff
@@ -1,38 +1,39 @@
 const {
   doc: {
-    builders: { join }
+    builders: { concat }
   }
 } = require('prettier/standalone');
 
+const indexed = node => (node.isIndexed ? ' indexed' : '');
+
+const visibility = node =>
+  node.visibility && node.visibility !== 'default'
+    ? concat([' ', node.visibility])
+    : '';
+
+const constantKeyword = node => (node.isDeclaredConst ? ' constant' : '');
+
+const storageLocation = node =>
+  node.storageLocation && node.visibility !== 'default'
```

Comment:
This is the only place where I think the code is a bit weird by asking whether `node.visibility !== 'default'` to make a choice for `storageLocation` but I took it from the original code.
What do you think @fvictorio @mattiaerre ??

Reply:
I think this makes sense, because `node.visibility === 'default'` means there's no explicit visibility in the original code.

Reply:
I agree @Janther why is visibility necessary to determine the location? it looks odd to me too. But it seems to work so I'll go ahead, approve and merge